### PR TITLE
update Vitis examples readme and doc

### PR DESCRIPTION
**Pull request by guotuofeng**
## Checklist before requesting a review
- [ ] Add unit tests for this change.
- [ ] Make sure all tests can pass.
- [ ] Update documents if necessary.
- [ ] Format your code by running `pre-commit run --all-files`
- [ ] Is this a user-facing change? If yes, give a description of this change to be included in the release notes.

### CODE REVIEW

File: examples/resnet/README.md [hunk 1]
```diff
@@ -15,20 +15,20 @@ The workflow in [resnet_static_ptq_cpu.json](resnet_static_ptq_cpu.json) is simi
 The workflow in [resnet_dynamic_ptq_cpu.json](resnet_dynamic_ptq_cpu.json) is similar to the above workflow, but specifically uses dynamic quantization instead of static/dynamic quantization.
 
 
-### ResNet optimization with Vitis-AI PTQ on CPU
-This workflow performs ResNet optimization on CPU with AMD Vitis-AI Quantization. It performs the optimization pipeline:
+### ResNet optimization with Vitis-AI PTQ on DPU
```
Comment (jambayk) [inline examples/resnet/README.md:18]: I think it should only be this line? the others should still be CPU.

File: docs/source/examples.md [hunk 2]
```diff
@@ -22,3 +20,6 @@
 
 # Acceleration on NPU
 [Inception model optimization on Qualcomm NPU](https://github.com/microsoft/Olive/tree/main/examples/snpe/inception_snpe_qualcomm_npu)
+
+# Acceleration on DPU
+[ResNet optimization with Vitis-AI Post Training Quantization](https://github.com/microsoft/Olive/tree/main/examples/resnet#resnet-optimization-with-vitis-ai-ptq-on-cpu)
```
Comment (jambayk) [inline docs/source/examples.md:25]: the link needs to end in `dpu` instead of `cpu`

File: examples/resnet/README.md (hunk 1 above)
Comment (guotuofeng) [reply examples/resnet/README.md:18]: good catch

File: examples/resnet/README.md [hunk 3]
```diff
@@ -15,8 +15,8 @@ The workflow in [resnet_static_ptq_cpu.json](resnet_static_ptq_cpu.json) is simi
 The workflow in [resnet_dynamic_ptq_cpu.json](resnet_dynamic_ptq_cpu.json) is similar to the above workflow, but specifically uses dynamic quantization instead of static/dynamic quantization.
 
 
-### ResNet optimization with Vitis-AI PTQ on CPU
-This workflow performs ResNet optimization on CPU with AMD Vitis-AI Quantization. It performs the optimization pipeline:
+### ResNet optimization with Vitis-AI PTQ on DPU
+This workflow performs ResNet optimization on DPU with AMD Vitis-AI Quantization. It performs the optimization pipeline:
 - *PyTorch Model -> Onnx Model -> AMD Vitis-AI Quantized Onnx Model*
 
 Config file: [resnet_vitis_ai_ptq_cpu.json](resnet_vitis_ai_ptq_cpu.json)
```
Comment (xiaoyu-work) [inline examples/resnet/README.md:22]: May I know the reason if it runs on DPU, why this config name is ending with cpu?

Comment (guotuofeng) [reply examples/resnet/README.md:22]: the optimization is run on CPU, but the output model should be targeted DPU.

File: docs/source/examples.md [hunk 4]
```diff
@@ -22,3 +20,6 @@
 
 # Acceleration on NPU
 [Inception model optimization on Qualcomm NPU](https://github.com/microsoft/Olive/tree/main/examples/snpe/inception_snpe_qualcomm_npu)
+
+# Acceleration on DPU
+[ResNet optimization with Vitis-AI Post Training Quantization](https://github.com/microsoft/Olive/tree/main/examples/resnet#resnet-optimization-with-vitis-ai-ptq-on-dpu)
```
Comment (EmmaNingMS) [inline docs/source/examples.md:25]: Suggest adding the HW info in the title "ResNet optimization with Vitis-AI Post Training Quantization for AMD DPU"

File: examples/resnet/README.md [hunk 5]
```diff
@@ -15,8 +15,8 @@ The workflow in [resnet_static_ptq_cpu.json](resnet_static_ptq_cpu.json) is simi
 The workflow in [resnet_dynamic_ptq_cpu.json](resnet_dynamic_ptq_cpu.json) is similar to the above workflow, but specifically uses dynamic quantization instead of static/dynamic quantization.
 
 
-### ResNet optimization with Vitis-AI PTQ on CPU
-This workflow performs ResNet optimization on CPU with AMD Vitis-AI Quantization. It performs the optimization pipeline:
+### ResNet optimization with Vitis-AI PTQ on DPU
```
Comment (EmmaNingMS) [inline examples/resnet/README.md:18]: Let's make it clearer. DPU -> AMD DPU

**Review by EmmaNingMS — CHANGES_REQUESTED** [review]

**Review by EmmaNingMS — APPROVED** [review]

**Pull request merged**